react-frontend/src/Authentication/Registration.js

```diff
@@ -53,12 +67,14 @@ const Registration = () => {
             onChange={handleChange}
           />
         </label>
-        <label htmlFor="password">
+        <label htmlFor="reenter-password">
```

Comment:
NIT: It may be nice if we maintain the same naming throughout the password confirmation component that is we keep confirm_password.